io/src/main/java/org/commonjava/maven/ext/io/PomIO.java

```diff
@@ -64,6 +67,9 @@
 @Singleton
 public class PomIO
 {
+    // TODO: Remove this if no side affects reported in 2022.
+    public static final String PARSE_POM_TEMPLATES = "parsePomTemplates";
```

Comment:
Should this be `public` or `private`? It currently seems only used within the class.

Reply:
No public - its used in a test

Reply:
Oh, there using the default public constructor? I think some version of Java is getting rid of those and they have to be made explicit.